Kconfig.xtos-dbg

```diff
@@ -12,3 +12,16 @@ config DEBUG_BLOCK_FREE
 	  already freed block of memory. Enabling this feature increases
 	  number of memory writes and reads, due to checks for memory patterns
 	  that may be performed on allocation and deallocation.
+
+config DEBUG_FORCE_COHERENT_BUFFER
+	bool "Force the allocator to allocate coherent buffer only"
+	default y
```

Comment:
deafult should be ```n``` atm otherwise we will end up with lost of performance bugs being reported.